src/encoded/tests/data/inserts/experiment_set_replicate.json

```diff
@@ -21,7 +21,7 @@
         "static_headers": [
             "item-page-headers.ExperimentSet.data-usage-guidelines",
             "release-updates.TEST1",
-            "help.analysis.file-formats.hic",
+            "resources.analysis.file-formats.hic",
```

Comment:
(Re: all insert changes-)
Moved some `Page` & `StaticSection` inserts over to 'Resources' NavItem/Dropdown, by setting as `children` (or children of children) of new insert for page with `name`/url `/resources`.